internal/pkg/cli/file/filetest/watchertest.go

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package filetest
```

Comment:
Do we use this?

Reply:
Yeah it's used in `run_local_test.go` but maybe the package name should be changed.